templates/careers/job-detail.html

```diff
@@ -45,73 +39,60 @@ <h1>Careers</h1>
-              <input id="{{ question.fields[0].name }}" name="{{ question.fields[0].name }}" type="text" {% if question.required %}required{% endif %} maxlength="255">
-              {% elif question.fields[0].type == "input_file" %}
-              <input id="{{ question.fields[0].name }}" name="{{ question.fields[0].name }}" type="file" {% if question.required %}required{% endif %} accept=".pdf, .doc, .docx, .txt, .rtf">
-              {% elif question.fields[0].type == "textarea" %}
-              <textarea id="{{ question.fields[0].name }}" name="{{ question.fields[0].name }}" type="textarea" {% if question.required %}required{% endif %}></textarea>
-              {% elif question.fields[0].type == "multi_value_single_select" %}
-              <select name="{{ question.fields[0].name }}" id="{{ question.fields[0].name }}" {% if question.required %}required{% endif %}>
-                <option value="" disabled="disabled" selected="">Select an option</option>
-                {% for answer_option in question.fields[0].get("values",[]) %}
-                <option value="{{ answer_option.value }}">{{ answer_option.label }}</option>
-                {% endfor %}
-              </select>
-              {% elif question.fields[0].type == "multi_value_multi_select" %}
-              <select name="{{ question.fields[0].name }}" id="{{ question.fields[0].name }}" multiple="" {% if question.required %}required{% endif %}>
-                <option value="" disabled="disabled">Select...</option>
-                {% for answer_option in question.fields[0].get("values",[]) %}
-                <option value="{{ answer_option.value }}">{{ answer_option.label }}</option>
-                {% endfor %}
-              </select>
-              {% endif %}
-            {% endfor %}
-          {% endif %}
-          <label for="location" class="is-required">Location (City)</label>
-          <input id="location" name="location" type="text" class="js-location" required maxlength="255">
-          <button type="button" class="js-locate-me">Locate me</button>
-          <span class="js-locate-error u-hide" style="color: #c7162b">Location unavailable</span>
-          <hr style="margin: 1rem 0;" />
-          <input type="submit" class="p-button--positive u-no-margin--bottom" name="submit" value="Submit application">
-        </fieldset>
-      </form>
+      {{ bleach.clean(job_content, tags=allowed_tags, strip=True) | safe }}
     </div>
   </div>
+
+  <div class="col-5 col-start-large-8 p-card--strip">
+    <h3>Apply for this role</h3>
+    <form action="{{ request.path }}" method="POST" enctype="multipart/form-data" class="js-roles-list--form" onsubmit="dataLayer.push({
+          'event': 'GAEvent',
+          'eventCategory': 'Form',
+          'eventAction': 'job application: {{ job.id }}',
+          'eventLabel': 'Submit Application',
+          'eventValue': undefined });" >
+      <fieldset style="border: none; margin-bottom: 0;">
```

Comment:
`u-no-margin--bottom`? I am not sure if we have something for border maybe we should?

Reply:
Ideally we would code up a true pattern in Vanilla but that will be in the future.